src/tscpp/util/ts_ip.cc

```diff
@@ -0,0 +1,121 @@
+/** @file
+
+    IP address handling support.
+
+    @section license License
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with this
+    work for additional information regarding copyright ownership.  The ASF
+    licenses this file to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+    License for the specific language governing permissions and limitations under
+    the License.
+*/
+
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <netdb.h>
+
+#include <tscore/ink_memory.h>
+
+#include "tscpp/util/ts_ip.h"
+
+namespace ts
+{
+IPAddrPair
+getbestaddrinfo(swoc::TextView name)
+{
+  if (swoc::IP4Addr addr; addr.load(name)) {
+    return {};
+  }
+
+  if (swoc::IP6Addr addr; addr.load(name)) {
+    return {};
+  }
```

Comment:
Why does this return empty if it is a valid v4 or v6 address? load returns true if the parse is successful, right?
http://docs.solidwallofcode.com/libswoc/reference/classswoc_1_1IP4Addr.html#abe02426da0f5f003cc45d81869eb3f3b

Reply:
Fixed.